scheduling/allocator.rb

```diff
@@ -339,6 +352,9 @@ def update(vm, vm_host)
           used_cpu_percent: Sequel[:used_cpu_percent] + vm.cpu_percent_limit,
           used_memory_gib: Sequel[:used_memory_gib] + vm.memory_gib
         )
+
+        # Update the host utilization
+        @vm_host_allocations.each { _1.update(vm, vm_host) }
```

Comment:
The vm now needs to be updated before hosts, otherwise this won't be correct anymore. that is a little fragile.

Reply:
> Are you saying it is fragile because this order is required, which is not evident from reading this method?
Yes. Adding a comment would be good.